packages/smooth_app/lib/pages/input/unfocus_when_tap_outside.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+
+/// Allows to unfocus TextField (and dismiss the keyboard) when user tap outside the TextField and inside this widget.
+/// Therefore, this widget should be put before the Scaffold to make the TextField unfocus when tapping anywhere.
+class UnfocusWhenTapOutside extends StatelessWidget {
+  const UnfocusWhenTapOutside({required this.child});
+
+  final Widget child;
+
+  @override
+  Widget build(BuildContext context) => GestureDetector(
```

Comment:
A `TapRegion` would be better here, has a `GestureDetector` is limited to the bound of the Widget.

Reply:
@g123k As the `child` is supposed to be a `Scaffold` (cf. comments) I think the bounds are large enough. 
For the record I just moved code when refactoring, I didn't write the original version, that's been working for the last 10 months.

Reply:
Ok, in that case, could you just rename it to `UnfocusFieldWhenTapOutside`, as it's a bit confusing, without reading the comment above (my fault)

Reply:
Will be fixed in https://github.com/openfoodfacts/smooth-app/pull/4916